console/src/main/java/org/eclipse/kapua/app/console/setting/ConsoleSettingKeys.java

```diff
@@ -17,6 +17,7 @@
 public enum ConsoleSettingKeys implements SettingKey {
     SKIN_RESOURCE_DIR("skin.resource.dir"), //
     LOGIN_BACKGROUND_CREDITS("login.background.credits"), //
+    LOGIN_GENERIC_SNIPPET("login.generic.snippet"), //
```

Comment:
It seems that `login.generic.snippet` key is not present in `src/main/resources/console-settings.properties`, thus failing the HTTP call when reading the value. I think we should explicitly set a dummy content, but insert the key anyway.